src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,73 @@
+package duke.command;
+
+import duke.core.Storage;
+import duke.core.TaskList;
+import duke.core.Ui;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+import java.time.LocalDate;
+
+/**
+ * Represents a command where a new task will be added to the TaskList during the usage of the Duke programme.
+ */
+public class AddCommand extends Command {
+
+    Task taskToAdd;
+
+    /**
+     * Private constructor that assigns the task to be added to this AddCommand.
+     * @param task the task to be assigned to this AddCommand
```

Comment:
Should there be no empty line between the Javadoc comment description and parameter section? I've noticed this issue in other Javadoc comments as well.